rpm/mosh.spec

```diff
@@ -1,21 +1,22 @@
 Name:		mosh
-Version:	1.2.5
+Version:	1.3.2
 Release:	1%{?dist}
 Summary:	Mobile shell that supports roaming and intelligent local echo
 
 License:	GPLv3+
 Group:		Applications/Internet
 URL:		https://mosh.org/
-Source0:	https://github.com/downloads/keithw/mosh/mosh-%{version}.tar.gz
+%undefine       _disable_source_fetch
+Source0:	https://github.com/mobile-shell/mosh/archive/refs/tags/%{name}-%{version}.tar.gz
 
 BuildRequires:	protobuf-compiler
 BuildRequires:	protobuf-devel
 BuildRequires:	libutempter-devel
 BuildRequires:	zlib-devel
 BuildRequires:	ncurses-devel
-BuildRequires:	openssl-devel
+BuildRequires:	openssl11-devel
```

Comment:
When testing this PR, this change gave me some problems.

In a CentOS 8.5 container, there is no `openssl11-devel` package (presumably because the `openssl-devel` package is itself version 1.1

In a CentOS 7.9 container, there is an `openssl11-devel` package if you enable EPEL, but then mosh's `./configure` can't find it (because `pkg-config --exists openssl` fails, since the package is called `openssl11`)